_src/pl/static/js/radio-replace.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Radio button replacement
+ * Handles functionality for the replacement radio buttons, which
+ * allows us to style them to our liking. Accessibility is main-
+ * tained via use of the default radio control. The replacement 
+ * control is not visible to screen readers.
+ */
+
+$(function() {
+
+    var CustomRadioReplacement = {
+
+        vars: {
+            replaced:       $('.replace-radio'),
+            replacedClass:  'is-replaced is-transparent',
+            customClass:    'wrapper-custom-radio',
+            wrapperClass:   'wrapper-replace-radio',
+            iconChecked:    'icon-dot-circle-o',
+            iconUnchecked:  'icon-circle-o'
+        },
+
+        init: function() {
+            this.replaceFoundRadios();
+            this.listenForRadioClick();
+        },
+
+        replaceFoundRadios: function() {
+            var variables = this.vars;
+
+            if (variables.replaced.length) {
+
+                variables.replaced.each(function(index, el) {
+                    var $el = $(el);
+
+                    $el.addClass(variables.replacedClass);
+                    $el.wrap('<div class="' + variables.wrapperClass + '"></div>');
+                    $el.parent('.' + variables.wrapperClass).append($('<div class="' + variables.customClass + '" aria-hidden="true"></div>'));
+                });
+
+                CustomRadioReplacement.setRadiosSelected();
+            }
+        },
+
+        listenForRadioClick: function() {
+            this.vars.replaced.on('click', function() {
+                CustomRadioReplacement.setRadiosSelected();
+            });
+        },
+
+        toggleSelectedState: function(state) {
+            return '<svg class="icon ' + state + '" title="Radio button"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/public/images/edx-svg/svgdefs.svg#' + state + '"></use></svg>';
+        },
+
+        setRadiosSelected: function() {
+            var variables = this.vars;
+
+            variables.replaced.each(function(index, el) {
+                var $el = $(el);
+
+                if ($el.is(':checked')) {
+                    $el.closest('.form-label')
+                        .addClass('is-selected');
+
+                    $el.next('.' + variables.customClass)
+                        .addClass('is-selected')
+                        .html(CustomRadioReplacement.toggleSelectedState(variables.iconChecked));
+                } else {
+                    $el.closest('.form-label')
+                        .removeClass('is-selected');
+
+                    $el.next('.' + variables.customClass)
+                        .removeClass('is-selected')
+                        .html(CustomRadioReplacement.toggleSelectedState(variables.iconUnchecked));
+                }
+            });
+        }
+
+    };
+
+    CustomRadioReplacement.init();
+});
```

Comment:
As with the dropdown code can you wrap this code in the revealing module pattern so only the init function is being exposed globally?